package.json

```diff
@@ -17,10 +17,10 @@
   "repository": "github:rgossiaux/svelte-headlessui",
   "version": "2.0.0",
   "scripts": {
-    "dev": "svelte-kit dev",
-    "build": "svelte-kit build",
-    "package": "svelte-kit package && find package/ -type f -exec sed -i '' -e 's/process.env.NODE_ENV/\"production\"/' {} \\;",
-    "preview": "svelte-kit preview",
+    "dev": "vite dev",
+    "build": "vite build",
+    "package": "svelte-package && find package/ -type f -exec sed -i '' -e 's/process.env.NODE_ENV/\"production\"/' {} \\;",
```

Comment:
The sveltekit library template [runs](https://github.com/sveltejs/kit/blob/b43ea97396f844064231b665977721e4de43a489/packages/create-svelte/templates/skeletonlib/package.template.json#L8) `publint` after `svelte-package`. FWIW, I ran `publint` on both this and the generated package.json and both are green.

Reply:
Thanks for the tip! Happy to add this tool to the workflow, I'm always paranoid about publishing something broken to npm.